rust/src/server/middleware/auth.rs

```diff
@@ -145,3 +153,33 @@ impl<'a, 'r> FromRequest<'a, 'r> for AuthInfo {
         };
     }
 }
+
+fn set_sentry_user(claims: &Claims, user_details: &UserDetails) {
+    let user_data: BTreeMap<String, Value> = vec![
+        (
+            "device_id".into(),
+            serde_json::json!(claims.deviceId.clone()),
+        ),
+        (
+            "First Name".into(),
+            serde_json::json!(user_details.firstName.clone()),
+        ),
+        (
+            "Last Name".into(),
+            serde_json::json!(user_details.lastName.clone()),
+        ),
+        (
+            "Phone number".into(),
```

Comment:
I feel that it might be dangerous to share the phone, first name and last name, could we leave only the device_id, and user_id?

Reply:
done!